### ISSUE TITLE
Feature Request Accessibility: Talkback Actions Menu

### ISSUE THREAD

**Issue by shortwavesurfer2009** [thunder-app/thunder#64]
I noticed on replies you can swipe one way to get reply and save while if you swipe the other direction you can upvote or downvote. Unfortunately a talkback user cannot easily perform these swipes. Talkback has a solution for this called the "actions" menu but i was not able to quickly find how to do it. I did run across [this Tusky issue](https://github.com/tuskyapp/Tusky/issues/1003) where it was done and figure it may be of some help.

**Comment by hjiangsu** [comment]
Hmm, yeah. One alternative that I thought of is to also allow for button actions on comments similar to posts. This would solve two problems:
- Enables these features for accessibility purposes
- Allows an alternative for users who dont particularly enjoy swipe gestures

**Comment by shortwavesurfer2009** [comment]
That would work too. Though i think the buttons should be hidden by default and shown with a long press of the comment.

**Comment by hjiangsu** [comment]
> Though i think the buttons should be hidden by default and shown with a long press of the comment.

I agree, it should be hidden by default (unless maybe specified in the settings)

**Comment by micahmo** [comment]
@shortwavesurfer2009 Would this issue be solved by #741? Or would you still like us to investigate the "TalkBack actions menu"?

**Comment by shortwavesurfer2009** [comment]
> @shortwavesurfer2009 Would this issue be solved by #741? Or would you still like us to investigate the "TalkBack actions menu"?

Yeah. #741
